pkg/daemon/config.go

```diff
@@ -137,6 +140,10 @@ func (output *ptp4lConf) populatePtp4lConf(config *string) error {
 				split := strings.IndexByte(line, ' ')
 				if split > 0 {
 					currentSection.options[line[:split]] = line[split:]
+					if (line[:split] == "masterOnly" && line[split:] == "0") ||
+						(line[:split] == "slaveOnly" && line[split:] == "1") {
+						hasSlaveInterfaceDefined = true
```

Comment:
slaveOnly is set on a per-config basis, not per-interface, so should rename hasSlaveInterfaceDefined

Reply:
updated `hasSlaveInterfaceDefined` => `hasSlaveConfigDefined`